addons/account/models/chart_template.py

```diff
@@ -195,7 +195,7 @@ def _load(self, template_code, company, install_demo):
         if install_demo and self.ref('base.module_account').demo and not reload_template:
             try:
                 with self.env.cr.savepoint():
-                    self._load_data(self._get_demo_data(company))
+                    self._load_data(self._get_demo_data(company), demo=True)
```

Comment:
```suggestion
                    self.sudo()._load_data(self._get_demo_data(company), demo=True)
```
That would not do the trick? (avoiding the extra param)

Reply:
Isn't it a bit risky ? The sudo() will propagate afterwards if I'm not wrong ?

Reply:
@clbr-odoo it would normally propagate within _load_data, but not after _load_data.  (so it would be more or less the same as with the current code?)

Reply:
> Isn't it a bit risky?

It's only for demo data so fine for me.

> The sudo() will propagate afterwards if I'm not wrong ?

Only inside the call of this function, which is fine. Anyway, it is exactly the same as what you are doing with the `demo` param in practice since the only thing that is not sudo'ed is manipulation of the data dicts (no database accesses).

Reply:
Okay, seems fair, thanks for the input. I fixed the commit.